homeassistant/components/screenlogic/config_flow.py

```diff
@@ -35,17 +35,22 @@ async def async_discover_gateways_by_unique_id(hass):
         return discovered_gateways
 
     for host in hosts:
-        mac = _extract_mac_from_name(host[SL_GATEWAY_NAME])
-        discovered_gateways[mac] = host
+        if host[SL_GATEWAY_NAME].startswith("Pentair:"):
```

Comment:
I think it would probably be enough to reject anything that doesn't start with `Pentair` as we want to try to be as passive as possible to devices we aren't going to setup

Reply:
Roger